packages/AutoTDD-Core.package/ATDDGitHubActionsClient.class/instance/requestOn..st

```diff
@@ -17,7 +17,11 @@ requestOn: aString
 	
 	^ response code
 		= 200
-		ifTrue: [ Json readFrom: response content readStream ]
+		ifTrue: [ 
+			(response contentType includesSubstring: 'application/json') 
+				ifTrue: [ Json readFrom: response content readStream ]
+				ifFalse: [ response content ]
+			]
```

Comment:
We should probably handle different `contentTypes` not only "Json" or "notJson":

```suggestion
		ifTrue: [ 
			(response contentType includesSubstring: 'application/json') 
				ifTrue: [ Json readFrom: response content readStream ]
			(response contentType includesSubstring: 'text/plain') 
				ifTrue: [ response content ]
			"throwSomeError (e.g. ContentTypeNotUnderstoodError)"
			]
```

Reply:
will do the else for 'text/plain' but let's do the exception handling some other time, because I am not really very sure how to do it exactly and we have to do exception handling either way for other stuff